tests/base_backend_tests.py

```diff
@@ -257,15 +258,20 @@ def test_BOT_ADMINS_unique_string(self):
 
 
 class TestExecuteAndSend(unittest.TestCase):
+    assets_path = os.path.join(os.path.dirname(__file__), 'assets')
+
     def setUp(self):
         self.dummy = DummyBackend()
         self.example_message = self.dummy.build_message('some_message')
         self.example_message.frm = self.dummy.build_identifier('noterr')
         self.example_message.to = self.dummy.build_identifier('err')
 
-        assets_path = os.path.join(os.path.dirname(__file__), 'assets')
-        templating.template_path.append(templating.make_templates_path(assets_path))
-        templating.env = templating.Environment(loader=templating.FileSystemLoader(templating.template_path))
+        duplicates_path = os.path.join(assets_path, 'duplicate_template')
+
+        self.tenv = templating.add_plugin_templates_path(assets_path)
```

Comment:
I've stopped fixing the test when I found that I'd have to reimplement all the existing logic to make it work.